src/traces/pie/calc.js

```diff
@@ -76,6 +75,9 @@ function calc(gd, trace) {
         }
     }
 
+    // Drop aggregate sums of value 0 or less
+    cd = cd.filter(function(elem) { return elem.v > 0; });
```

Comment:
```suggestion
    cd = cd.filter(function(elem) { return elem.v >= 0; });
```

Reply:
So that `funnelarea_line_width` mock render as expected.

Reply:
Done